sktime/networks/lstmfcn.py

```diff
@@ -0,0 +1,128 @@
+# -*- coding: utf-8 -*-
+"""LSTM-FCN Network (minus the final output layer)."""
+
+__author__ = ["Jack Russon", "AurumnPegasus"]
+
+from sktime.networks.attentionlstm import _getAttentionLSTM
+from sktime.networks.base import BaseDeepNetwork
+from sktime.utils.validation._dependencies import _check_dl_dependencies
+
+_check_dl_dependencies(severity="warning")
+
+
+class LSTMFCNNetwork(BaseDeepNetwork):
+    """Establish the network structure for a LSTM-FCN.
+
+    Adapted from the implementation used in [1]
+
+    Parameters
+    ----------
+    kernel_sizes     : tuple, default = (8, 5, 3)
+        specifying the length of the 1D convolution window
+    filter_sizes   : tuple, default = (128, 256, 128)
+        size of filter for each conv layer
+    num_cell        : int, default = 8,
+        number of cells in LSTM layer
+    dropout      : float, default = 0.8
+        dropout for the network
+    random_state    : int, default = 0
+        seed to any needed random actions
+
+    Notes
+    -----
+    Implementation of LSTMFCNClassifier from Karim et al (2019).
+
+    References
+    ----------
+    .. [1] @article{Karim_2019,
```

Comment:
Please make the format of article reference the same as you made on line 52 in lstmfcn.py

Reply:
Seems like I had made a mistake in referencing as well!
Wrote the references in the format needed, let me know if its cool now.